frontend/src/App.jsx

```diff
@@ -41,6 +42,7 @@ const theme = createTheme({
 const App = () => (
     <ThemeProvider theme={theme}>
         <div className={classes.container}>
+            <HeaderController />
```

Comment:
We won't be using this header on all pages so it should be removed from here, any page that needs it will import it